raster/r.texture/testsuite/test_texture.py

```diff
@@ -24,59 +25,164 @@ def setUpClass(cls):
-        self.assertRasterMinMax(
-            map=var_output,
-            refmin=0,
-            refmax=9843.07,
-            msg="var_Var in degrees must be between 0.74 and 9843.07",
-        )
+    def test_sv(self):
+        """Testing method sv"""
+        sa_SA = "sv_SV"
+        values = """min=0
+        max=45368492
+        mean=2248724.35829364
+        variance=2332049431762.5
+        cells=1000246"""
+        self.assertModule("r.texture", input=self.input, output="sv", method="sv")
+        self.assertRasterFitsUnivar(sa_SA, reference=values, precision=2)
 
-    def test_idm(self):
-        """Testing method idm"""
-        idm_IDM = "idm_IDM"
-        self.assertModule("r.texture", input=self.input, output="idm", method="idm")
-        self.assertRasterMinMax(
-            map=idm_IDM,
-            refmin=0,
-            refmax=9843.07,
-            msg="idm_IDM_IDM in degrees must be between 0.74 and 9843.07",
-        )
+    def test_se(self):
+        """Testing method se"""
+        sa_SA = "se_SE"
+        values = """min=0
+        max=2.29248
+        mean=1.99163
+        variance=0.025583
+        cells=1000246"""
+        self.assertModule("r.texture", input=self.input, output="se", method="se")
+        self.assertRasterFitsUnivar(sa_SA, reference=values, precision=2)
+
+    def test_entr(self):
+        """Testing method entr"""
+        sa_SA = "entr_Entr"
+        values = """min=-0
+        max=3.29248
+        mean=3.20826
+        variance=0.0106857
+        cells=1000246"""
+        self.assertModule("r.texture", input=self.input, output="entr", method="entr")
+        self.assertRasterFitsUnivar(sa_SA, reference=values, precision=2)
+
+    def test_dv(self):
+        """Testing method dv"""
+        sa_SA = "dv_DV"
+        values = """min=-4.33281e-05
+        max=0.115451
+        mean=0.00110391
+        variance=2.37569e-06
+        cells=1000246"""
+        self.assertModule("r.texture", input=self.input, output="dv", method="dv")
+        self.assertRasterFitsUnivar(sa_SA, reference=values, precision=2)
+
+    def test_de(self):
+        """Testing method de"""
+        sa_SA = "de_DE"
+        values = """min=-0
+        max=2.29248
+        mean=1.61153
+        variance=0.0550566
+        cells=1000246"""
+        self.assertModule("r.texture", input=self.input, output="de", method="de")
+        self.assertRasterFitsUnivar(sa_SA, reference=values, precision=2)
+
+    def test_moc1(self):
+        """Testing method moc1"""
+        sa_SA = "moc1_MOC-1"
+        values = """min=-0.910445
+        max=0
+        mean=-0.78294
+        variance=0.00736236
+        cells=1000246"""
+        self.assertModule("r.texture", input=self.input, output="moc1", method="moc1")
+        self.assertRasterFitsUnivar(sa_SA, reference=values, precision=2)
+
+    def test_moc2(self):
+        """Testing method moc2"""
+        sa_SA = "moc2_MOC-2"
```

Comment:
This is confusing. I realize this comes from the original text and also the difficulty that r.texture generates its own names from a basename, but can you do this:

```python
basename = "..."
method = "..."
output = f"{basename}_{method}"
self.assertModule("r.texture", input=self.input, output=basename, method=method)
self.assertRasterFitsUnivar(output, reference=values, precision=2)
```

or simply without variable looks okay too given that the code is short:

```python
self.assertModule("r.texture", input=self.input, output="basename", method="method")
self.assertRasterFitsUnivar("basename_method", reference=values, precision=2)
```

Reply:
I added `basename`, `method`, and `output`. Thanks for the suggestions.